core/src/main/kotlin/org/lflang/generator/cpp/CppStandaloneCmakeGenerator.kt

```diff
@@ -153,6 +153,8 @@ class CppStandaloneCmakeGenerator(private val targetConfig: TargetConfig, privat
                 |cmake_minimum_required(VERSION 3.5)
                 |project(${fileConfig.name} VERSION 0.0.0 LANGUAGES CXX)
                 |
+                |option(REACTOR_CPP_LINK_EXECINFO "Link against execinfo" OFF)
```

Comment:
Why do we need to link to execinfo?

Reply:
It is necessary to do this explicitly on Alpine. There is some discussion [here](https://github.com/awslabs/aws-lambda-cpp/issues/124), in particular [this comment](https://github.com/awslabs/aws-lambda-cpp/issues/124#issuecomment-974645808).